src/Dev/FocusPointMigrationTask.php

```diff
@@ -49,36 +50,72 @@ protected function changeDbFields($from, $to, $message)
 
         // Safety net
         if (!isset($fields["{$to}X"]) || !isset($fields["{$to}Y"])) {
-            throw new \Exception("$imageTable table does not have \"{$to}X\" and \"{$to}Y\" fields. Did you run dev/build?");
+            throw new \Exception("{$imageTable} table does not have \"{$to}X\" and \"{$to}Y\" fields. Did you run dev/build?");
         }
 
         // Update all Image tables
         $imageTables = [
             $imageTable,
-            $imageTable . "_" . Versioned::LIVE,
-            $imageTable . "_Versions",
+            $imageTable . '_' . Versioned::LIVE,
```

Comment:
Are the quote changes a PSR thing or just personal preference?

Reply:
another php-cs-fixer change